Kernel/Syscalls/sched.cpp

```diff
@@ -27,6 +27,9 @@ ErrorOr<NonnullRefPtr<Thread>> Process::get_thread_from_pid_or_tid(pid_t pid_or_
         peer = Thread::current();
         if (pid_or_tid != 0)
             peer = Thread::from_tid(pid_or_tid);
+        // Only superuser can access other processes' threads.
```

Comment:
This looks logically separate from finding the thread, so this could use a newline above the comment to make it not too crowded.